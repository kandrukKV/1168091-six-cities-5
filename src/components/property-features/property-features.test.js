import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import PropertyFeatures from "./property-features";

it(`PropertyFeatures is rendered correctly`, () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(
      <PropertyFeatures
        type={`type`}
        bedrooms={1}
        adults={2}
      />
  );
  expect(tree).toMatchSnapshot();
});
