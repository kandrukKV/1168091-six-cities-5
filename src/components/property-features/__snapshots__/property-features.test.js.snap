// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyFeatures is rendered correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    type
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    1
     Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    2
     adults
  </li>
</ul>
`;
